usaspending_api/etl/es_transaction_mapping.json

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
Why is this a new file instead of modifying the existing file?

Reply:
Correct, we are currently using `es_transaction_template.json` for the index settings. The ETL process is currently using a template to guarantee that these settings are applied to a new index. This new mapping file is used by the python library when creating a new index. AFAIK there is only one difference in structure between the two files, which is the `index_patterns` key. If @miahunsicker  wants to add a new file then I would request `es_transaction_template.json` is updated to match this new file.